apps/portal/src/app/shared/components/notification-filters/notification-filters.html

```
@@ -0,0 +1,181 @@
+﻿<!-- GitLab-style unified search + filter bar -->
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)